model/metrics/v8js-metrics.yaml

```diff
@@ -0,0 +1,92 @@
+groups:
+  - id: metric.veightjs.gc.duration
+    type: metric
+    metric_name: v8js.gc.duration
+    brief: "Garbage collection duration."
+    instrument: histogram
+    unit: "s"
+    stability: experimental
+    prefix: v8js.gc
+    attributes:
+      - ref: v8js.gc.type
+        requirement_level: recommended
+
+  - id: metric.veightjs.memory.total
+    type: metric
+    metric_name: v8js.memory.total
+    brief: "Total memory size."
```

Comment:
To what "total size" here refers to? Is it the total size it has used? it is the total size available? I think this needs a better explanation.